doc/netplan.md

```diff
@@ -437,6 +437,24 @@ similar to ``gateway*``, and ``search:`` is a list of search domains.
             dhcp6: true
             optional-addresses: [ ipv4-ll, dhcp6 ]
 
+``activation-mode`` (scalar) – since **0.102**
```

Comment:
This is now going to be v0.103
```suggestion
``activation-mode`` (scalar) – since **0.103**
```

Additionally, we might choose a more generic name here, like `interface-mode`, in order to support an additional mode: `ignore`, where netplan should not touch the interface at all, but leave it up to the administrator to configure that interface manually. ([LP: #1918198](https://bugs.launchpad.net/netplan/+bug/1918198))